extension/war/src/main/webapp/WEB-INF/conf/social-extension/social/metadata-plugins-configuration.xml

```diff
@@ -100,6 +100,29 @@
         </object-param>
       </init-params>
     </component-plugin>
+    <component-plugin>
+      <name>AttachmentMetadataPlugin</name>
+      <set-method>addMetadataTypePlugin</set-method>
+      <type>org.exoplatform.social.metadata.MetadataTypePlugin</type>
+      <init-params>
+        <value-param>
+          <name>allowMultipleItemsPerObject</name>
+          <description>Whether to allow adding the same object to the same Metadata or not</description>
+          <value>true</value>
+        </value-param>
+        <object-param>
+          <name>metadataType</name>
+          <object type="org.exoplatform.social.metadata.model.MetadataType">
+            <field name="id">
+              <int>7</int>
+            </field>
+            <field name="name">
+              <string>attachment</string>
```

Comment:
We may need to add it as plural since it will reference a list of files.

```suggestion
              <string>attachments</string>
```

Reply:
done!